packages/apollo-federation/src/service/printFederatedSchema.ts

```diff
@@ -1,87 +1,119 @@
-/*
```

Comment:
Reviewer: this is a copy/paste of `newPrintFederatedSchema.ts` from previous commits. Nothing from this file change is new code - this is the drop-in update for the schema printer.